WalletWasabi.Fluent/Helpers/NotificationHelpers.cs

```diff
@@ -49,7 +52,7 @@ public static void Show(object viewModel)
 		NotificationManager?.Show(viewModel);
 	}
 
-	private static bool TryGetNotificationInputs(ProcessedResult result, [NotNullWhen(true)] out string? message)
+	private static bool TryGetNotificationInputs(Wallet wallet, ProcessedResult result, [NotNullWhen(true)] out string? message)
```

Comment:
It seems it doesn't require the whole wallet class, but only the `UsdExchangeRate`